sys/src/cmd/frontusb/cam/ctl.c

```diff
@@ -216,7 +216,7 @@ penumwrite(Cam *c, int term, Param *p, char **f, int nf)
 }
 
 char *
-pformatread(Cam *c, int, Param *)
+pformatread(Cam *c, int term, Param *param)
```

Comment:
This is an unfortunate spatch thing we can't fix, but it's a shame we can't say, Rust style, _term, and _param to indicate we won't use these.

In other parts of the tree we do this to indicate unused args.

Reply:
This was another manual change - detected by gcc, fixed by hand.  As such, we can precede with underscore.